Add Ceres products to SORA docs (#225)
sora-xor/sora-docs: src/ceres/governance.md

---
title: "Governance | SORA Docs"
head:
  - - meta
    - name: description
      content: "Ceres Governance platform is an important step towards decentralization"
  - - meta
    - name: keywords
      content: "Ceres, DeFi, governance, Polkaswap, SORA network"
---

# Ceres Governance platform

> [Ceres Governance platform](https://dapps.cerestoken.io/governance) is an important step towards decentralization. 

Reviewer: ```suggestion
The [Ceres Governance platform](https://dapps.cerestoken.io/governance) is an essential step towards decentralization. 
```

> Ceres token holders are able to influence decisions concerning the project such as proposing or deciding on new feature proposals.

Reviewer: ```suggestion
Ceres token holders can influence decisions concerning the project, such as proposing or implementing new features.
```
